src/wallet/rpcwallet.cpp

```diff
@@ -2383,9 +2382,10 @@ UniValue walletpassphrase(const JSONRPCRequest& request)
     if (nSleepTime < 0) {
         throw JSONRPCError(RPC_INVALID_PARAMETER, "Timeout cannot be negative.");
     }
-    // Clamp timeout to 2^30 seconds
-    if (nSleepTime > (int64_t)1 << 30) {
-        nSleepTime = (int64_t)1 << 30;
+    // Clamp timeout
+    int64_t max_sleep_time = 100000000; // larger values trigger a macos/libevent bug?
```

Comment:
I think we currently use `constexpr` (without `static`) and UPPER_CASE for compile time constants.